InTheMiddle2/InTheMiddle2/TableViewController.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  TableViewController.swift
+//  InTheMiddle2
+//
+//  Created by Justin Abarquez on 5/3/18.
+//  Copyright © 2018 Justin Abarquez. All rights reserved.
+//
+
+import UIKit
+
+class TableViewController: UITableViewController {
+    var restaurants: [String] = ["In-N-Out", "Seniore's Pizza", "Cafe Rosso"]
+    var addresses: [String] = ["123 Burgertown", "33 Pizza Lane", "San Francisco State University, 1600 Holloway Ave, San Francisco, CA 94132"]
+    
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return restaurants.count
+    }
+    
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        
+        tableView.rowHeight = UITableViewAutomaticDimension
+        
+        let restaurant = restaurants[indexPath.row]
+        let address = addresses[indexPath.row]
+    
+        let cell = TableViewCell(style: UITableViewCellStyle.default, reuseIdentifier: "restaurantCell")
+        
+        cell.labRestaurant.text = restaurant
+        cell.labRestaurant.font = UIFont(name: "Avenir", size: 22)
+        cell.labAddress.numberOfLines = 0
+        cell.labRestaurant.lineBreakMode = NSLineBreakMode.byWordWrapping
```

Comment:
Maybe you can add label text into the story board and reference that to limit constraints of text on the screen for table view. This is be easier to manage with alignment issues and constraints inside the UI table view.